src/app/pages/loading-waiting-states/progress-bar-indeterminate/progress-bar-indeterminate.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar data-cy="pxb-toolbar" flexLayout="row" class="app-bar mat-elevation-z2" [color]="checked ? 'primary' : ''">
+    <button data-cy="toolbar-menu" *ngIf="isSmall" mat-icon-button (click)="toggleMenu()" style="margin-left: -8px">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Progress Bars (Indeterminate)</div>
+    <mat-progress-bar mode="indeterminate"></mat-progress-bar>
+    <pxb-spacer></pxb-spacer>
+    <mat-slide-toggle labelPosition="before" [checked]="checked" (change)="onToggle($event)">Use Dark Theme</mat-slide-toggle>
```

Comment:
Use dark theme should be a global functionality — i.e., toggling the button in one page would change the overall application state

Reply:
On mobile, the toggle would become a menu item — see the leftmost screen in Figma

Reply:
I am happy to leave this functionality out for this round though. We can revisit this in the future.